open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1616,12 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    NextPageToken:
+      description:
+        An opaque next page token, when non-null and non-empty this indicates that more results can be returned by server.
```

Comment:
```suggestion
        An opaque next page token, indicating that more results can be returned by the server. The value of NextPageToken must be provided via the pageToken query parameter in the following request to fetch the next page.
```

Reply:
I believe ive included this in the pageToken section
```
    page-token:
      name: pageToken
      in: query
      description:
        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
        that they would prefer the response to be paginated.
        
        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
        For servers that support pagination, they will recognize pageToken and return a NextPageToken in response if there are more results available.
        After initial request, it is expected that the NextPageToken from the last response be used as the pageToken in the subsequent request.
        For servers that do not support pagination, they will ignore the pageToken and return all results.

```

Reply:
can we make the python documentation the same as the OpenAPI description?

Reply:
Actually, why do we have a definition of page token and next page token? I think this should just be `PageToken`, and the page-token query parameter schema should point to this, so does the `next-page-token` fields in the response. What do you think?